sklearn/metrics/ranking.py

```diff
@@ -184,23 +185,36 @@ def _binary_average_precision(y_true, y_score, sample_weight=None):
                                  average, sample_weight=sample_weight)
 
 
-def roc_auc_score(y_true, y_score, average="macro", sample_weight=None):
+def roc_auc_score(y_true, y_score, multiclass="ovr", average="macro",
+                  sample_weight=None):
     """Compute Area Under the Curve (AUC) from prediction scores
 
-    Note: this implementation is restricted to the binary classification task
-    or multilabel classification task in label indicator format.
-
     Read more in the :ref:`User Guide <roc_metrics>`.
 
     Parameters
     ----------
     y_true : array, shape = [n_samples] or [n_samples, n_classes]
         True binary labels in binary label indicators.
+        The multiclass case expects shape = [n_samples] and labels
+        with values from 0 to (n_classes-1), inclusive.
 
     y_score : array, shape = [n_samples] or [n_samples, n_classes]
         Target scores, can either be probability estimates of the positive
         class, confidence values, or non-thresholded measure of decisions
         (as returned by "decision_function" on some classifiers).
+        The multiclass case expects shape = [n_samples, n_classes]
+        where the scores correspond to probability estimates.
+
+    multiclass : string, ['ovr' (default), 'ovo']
+        Note: multiclass ROC AUC currently only handles the 'macro' and
+        'weighted' averages.
+
+        ``'ovr'``:
```

Comment:
I kept this the same because I wanted it to be uniform with the [documentation a few lines below](https://github.com/scikit-learn/scikit-learn/pull/7663/files#diff-7da4afad051915b763846995395fbda3L215). Do you have a preference for whether I should just change both to the format you suggested?